### ISSUE TITLE
Ability to rearrange tabs

### ISSUE THREAD

**Issue by Blackbaud-ToddRoberts** [blackbaud/skyux-tabs#19]
When tabs can be added/closed, user should be able to rearrange the tab order.

https://blackbaud.invisionapp.com/share/7XO2FUJ5RHG#/screens/320047108

Approved
